client/settings/general-settings/index.js

```diff
@@ -13,12 +13,17 @@ import { useDevMode, useIsWCPayEnabled, useTestMode } from 'wcpay/data';
 import CardBody from '../card-body';
 import InlineNotice from 'wcpay/components/inline-notice';
 import SetupLivePaymentsModal from 'wcpay/overview/modal/setup-live-payments';
+import DevModeConfirmationModal from './dev-mode-confirm-modal';
 
 const GeneralSettings = () => {
 	const [ isWCPayEnabled, setIsWCPayEnabled ] = useIsWCPayEnabled();
 	const [ isEnabled, updateIsTestModeEnabled ] = useTestMode();
 	const [ modalVisible, setModalVisible ] = useState( false );
 	const isDevModeEnabled = useDevMode();
+	const [
+		devConfirmationModalVisible,
+		setDevConfirmationModalVisible,
```

Comment:
I think this would be `testMode`, not `devMode`. And we can reduce the length of variable names by removing `Confirmation`. 
```suggestion
		testModeModalVisible,
		setTestModeModalVisible,
```

Reply:
Done